lib/r10k/source/git.rb

```diff
@@ -109,11 +115,27 @@ def desired_contents
     environments.map {|env| env.dirname }
   end
 
+  def filter_branches(branches, ignore_prefixes)
+    filter = Regexp.new("^(#{ignore_prefixes.join('|')})")
```

Comment:
This looks pretty good. I think `Regexp.new("^#{Regexp.union(ignore_prefixes)}")` is equivalent, but I'm not sure it's any cleaner.

Reply:
It's maybe a little clearer (maybe).